apps/system/js/haiku_emoji_panel.js

```diff
@@ -8,6 +8,16 @@
   };
   var mozMessage = navigator.mozMobileMessage;
   var logStatus = document.querySelector('#emoji-status-bottom');
+  var handleNextSMS = true;
```

Comment:
emojis.removeEventListener('click', this.emojiClickHandler);
setTimeout(() => {
  emojis.addEventListener('click', this.emojiClickHandler);
}, 2000);